### PR TITLE
prop hunt ss14 (real)

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
adds chameleon projector to the uplink for 7tc

lets you click any item/anchorable thing and change your sprite/name to it

if you get picked up then you revert back to human, which you can do manually using the action. same if you take damage, but then your player also take however much damage the disguise did. once you take enough damage to "kill" the disguise it automatically reverts. 
heavier props can take more hits but 1. you get no damage reduction 2. if you take over 100 damage you instantly crit.

you also have actions to disable rotation and anchor in place for better hiding

Requires space-wizards/RobustToolbox#5022

## Why / Balance
run into maints then become a crowbar and hide somewhere to avoid capture :trollface:
play honour system prop hunt if you revert you lose :trollface:
maybe someone could use it to make actual prop hunt gamemode some day, make it delete instead of revert when disguise is killed
hunters get a single china lake shell, drozd, kammerer, inspector, crowbar :trollface:

## Technical details
polymorph and some serializationmanager trolling :trollface:

## Media

ketchup (solution visuals fixed now)

https://github.com/space-wizards/space-station-14/assets/39013340/3e5990fc-e98c-4546-adcb-4f11af1b7d57

top: real
bottom: props <ins>/!\\</ins> (notice the left baton has its on appearance copied correctly)

status icons cant rat you out

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
no

**Changelog**
:cl:
- add: The Chameleon Projector has been added to the uplink for 7 TC, it lets you disguise as anything (within reason).

### CODE REVIEW

**Comment by beck-thompson** [comment]
So cool! Is it only a one time use thing or can you re-use it after being caught?

**Comment by deltanedas** [comment]
you can reuse it as long as you dont die, in a bit ill make it so when you take enough damage (proportional to prop mass) you take it in real life

**Comment by Hmeister-real** [comment]
cant wait for sec to start frantically picking up every trash object in disposals searching for a syndicate escapee :trollface:

**Comment by K-Dynamic** [comment]
Chameleon project as a corpse or a nukie balloon :trollface:

**Comment by beck-thompson** [comment]
I tried to check this out myself but I kept getting build errors. What happens when someone blows a whistle when you are in prop mode? Does it not show anything, show the "!" or take you out and show the "!"?

**Comment by deltanedas** [comment]
> I tried to check this out myself but I kept getting build errors.

it needs engine change so apply it locally or wait for merge + submodule update pr

>  What happens when someone blows a whistle when you are in prop mode? Does it not show anything, show the "!" or take you out and show the "!"?

will check

**Comment by ps3moira** [comment]
If someone has a security hud or your suits coords are on, will you be outted?

**Comment by deltanedas** [comment]
- havent checked :trollface:
- iirc polymorph sends original body to nullspace so you just vanish from sensors

**Comment by KingOnBudget** [comment]
its such a cool item but 7tc for this is too expensive imo.

**Comment by Hmeister-real** [comment]
> its such a cool item but 7tc for this is too expensive imo.

its an infinite use get-away tool though, just run into maintanance, open a locker, become an item, and boom your gone unless sec is robust or you're sloppy

**Comment by mirrorcult** [comment]
based

**Comment by potato1234x** [comment]
Does this change your bounding box too

**Comment by deltanedas** [comment]
no or cigarette butt ultra troll

**Comment by metalgearsloth** [comment]
"Depends on" isn't valid syntax for engine PRs last I checked https://github.com/space-wizards/submodule-dependency/

**Comment by deltanedas** [comment]
> You can also use "Depends on" with the same format.

also i had no idea this was even a thing i just did it for clarity

**Comment by 2013HORSEMEATSCANDAL** [comment]
Could I use this to add the morph as a midround antag ?

**Comment by EmoGarbage404** [comment]
i feel like using polymorph for this is really weird and it should just use appearance copying of some kind

**Comment by deltanedas** [comment]
- how it already copies appearance is sus enough, having to be able to revert back to whatever it was before would be a nightmare
- having to manually disable all hands, inventory items, implants, learned spells, etc when disguised would be horrible and prone to breaking
- polymorph just werk

**Comment by SoulFN** [comment]
Nukeops that disguised themselves as a nuke

**Comment by superjj18** [comment]
> Nukeops that disguised themselves as a nuke 

\>be warops team
\>go to ats and wait for a purchase to arrive
\>open box turn yourselves into whatever the contents is
\>be delivered into cargo
\>preform the funni 
\>cargo conquered instantly

Trojan horse ops

**Comment by deltanedas** [comment]
merg

**Review by mirrorcult — APPROVED** [review]

File: Resources/Textures/Objects/Devices/chameleon_projector.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at ",
```
Comment (deltanedas) [inline Resources/Textures/Objects/Devices/chameleon_projector.rsi/meta.json:4]: brooo how did nobody spot this copy paste fail

**Pull request merged**